src/sw/redis++/reply.h

```diff
@@ -47,7 +47,7 @@ template <typename T>
 struct ParseTag {};
 
 template <typename T>
-inline T parse(redisReply &reply) {
+T parse(redisReply &reply) {
```

Comment:
Why remove `inline`? Since the function body is quite simple, I'd like the compiler to inline it.

Reply:
It's implicitly inline, but I'll add `inline` back.